baybe/searchspace/core.py

```diff
@@ -380,6 +380,24 @@ def transform(
 
         return comp_rep
 
+    @property
+    def constraints_augmentable(self) -> tuple[Constraint, ...]:
```

Comment:
i dont prefer the human readable verison in such instances because imagine we have `constraints` and 5 subsets of it, they would all poten tially start with a different letter...

instead I prefer `constraints_subset1`, `constraints_subset_2` etc

this is similar to folder structure, which is just logical